doc/conf.py

```diff
@@ -12,9 +12,8 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
-import os
-sys.path.insert(0, os.path.abspath('..'))
+from libnacl import __version__ as version
```

Comment:
This requires libnacl to be installed to generate the documentation. The `sys.path.insert` allows the docs to be built without being installed...